frontend/src/containers/UserListing.tsx

```diff
@@ -167,9 +167,6 @@ function UserListing() {
               buttonText={t("UserListingDropdownMenuActions")}
               variant="outline"
             >
-              <MenuItem onSelect={changeRole} disabled={selected.length === 0}>
```

Comment:
@DaveOrDavid I don't think you meant to delete this block of code.  I think your code was not sync'ed (?)

Reply:
Did a rebase prior to working so a mistake by me on removing this from merge conflicts. I'll change and resubmit